### ISSUE TITLE
Migrate to youtube v3 for pulling previous video details in the dropdown

### ISSUE THREAD

**Issue by srish** [unhangout/unhangout#433]

**Comment by srish** [comment]
This issue was added in connection with a bug that was causing the youtube videos to not render in the previous video list properly. It was addressed here https://github.com/unhangout/unhangout/pull/416. Closing this for now.

**Comment by thehunmonkgroup** [comment]
I don't believe this issue should be closed. #416 is an ugly hack, not a long-term fix!
